dev-tools/validation/check-adr-review-status.py

```
@@ -0,0 +1,371 @@
+) -> None:
+    disposition = fields.get("Human review disposition")
+    allowed = STATUS_TO_HUMAN_REVIEW_DISPOSITIONS[status]
+    if disposition not in allowed:
+        fail(
+            f"{context}: ADR status {status!r} does not allow human review "
+            f"disposition {disposition!r}; allowed dispositions are "
+            f"{sorted(allowed)}"
+        )
+
+
+def checked_reviews(path: Path) -> dict[int, list[Review]]:
```

Ruff (0.16.0)

[warning] Too many branches (13 > 12)

(PLR0912)